basemaps/localization.md

```diff
@@ -66,133 +116,97 @@ Extending our Switzerland example with exonym and endonym from other languages:
-
-## Common languages by country
-
-The following country and international organizations worldviews are supported:
-
-| Country | Native name | Common language | Localized `name:*` value | Recommended `name:*` pairing |
-|---------|-------------|------------------|--------------------------|--------------------------|
-| Argentina | Argentina | Spanish | `name:es` | `name:it`, `name:fr`, `name:en`, `name:de` |
-| Bangladesh | বাংলাদেশ | Bengali | `name:bn` | _n/a_ |
-| Brazil | Brasil | Portugese | `name:pt` | `name:es`, `name:it`, `name:fr`, `name:en`, `name:de` |
-| China | 中国 | Chinese | `name:zh-Hans` | `name:zh`, `name:zh-Hant` |
-| Egypt | مصر | Arabic | `name:ar` | `name:fr`, `name:en`, `name:de` |
-| France | France | French | `name:fr` | `name:es`, `name:it`, `name:pt`, `name:en`, `name:de` |
-| Germany | Deutschland | German | `name:de` | `name:en`, `name:fr`, `name:es`, `name:it` |
-| Greece | Ελλάς | Greek | `name:el` | _n/a_ |
-| India | भारत | Hindi and many other | `name:hi`, +++ | `name:en` |
-| Indonesia | Indonesia | Indonesian | `name:id` | |
-| Israel | ישראל | Hebrew | `name:he` | _n/a_ |
-| Italy | Italia | Italian | `name:it` | `name:es`, `name:fr`, `name:pt`, `name:en`, `name:de` |
-| Japan | 日本 | Japanese | `name:ja` | _n/a_ |
-| Morocco | المغرب | Arabic | `name:ar` | `name:fr`, `name:en`, `name:de` |
-| Nepal | नेपाल | Nepalese | `name:ne` | `name:en`|
-| Netherlands | Nederland | Dutch | `name:nl` | `name:en`, `name:de`, `name:fr`, `name:es`, `name:it`  |
-| Pakistan | پاکستان | Urdu | `name:ur` | _n/a_ |
-| Palestine | فلسطين | Arabic | `name:ar` | _n/a_ |
-| Poland | Polska | Polish | `name:pl` | `name:de`, `name:en` |
-| Portugal | Portugal | Portugese | `name:pt` | `name:es`, `name:it`, `name:fr`, `name:en`, `name:de` |
-| Russia | Россия | Russian | `name:ru` | _n/a_ |
-| Saudi Arabia | المملكة العربية السعودية | Arabic | `name:ar` | _n/a_ |
-| South Korea | 한국 | Korean | `name:ko` | _n/a_ |
-| Spain | España | Spanish | `name:es` | `name:pt`, `name:it`, `name:fr`, `name:en`, `name:de` |
-| Sweden | Sverige | Swedish | `name:sv` | `name:en` |
-| Taiwan | 中華民國 | Traditional Chinese | `name:zh-Hant` | `name:zh-Hans`, `name:zh`|
-| Turkey | Türkiye | Turkish | `name:tr` | `name:fr`, `name:en`, `name:de` |
-| Ukraine | Україна | Ukrainian | `name:uk` | `name:ru` |
-| United Kingdom | United Kingdom | English, Welsh, Scottish, Irish, others | `name:en` | `name:es`, `name:fr`, `name:en`, `name:de` |
-| United States | United States | English, Spanish, French, others | `name:en` | `name:es`, `name:fr`, `name:en`, `name:de` |
-| Vietnam | Việt Nam | Vietnamese | `name:vi` | `name:fr`, `name:en`, `name:es`, `name:de` |
+
+### List of Supported Languages
+
+| Language | Native Name | `name:*` Tag | Script |
+| ----- | ----- | ----- | ----- |
+| Arabic | اَلْعَرَبِيَّةُ | `name:ar` | `Arabic` |
+| Bulgarian | български | `name:bg` | `Cyrillic` |
+| Chinese (Simplified) | 中文 汉语 | `name:zh-Hans` | `Han` |
```

Comment:
Are we exporting `zh` or differentiating `name:zh-Hans` and `name:zh-Hant`?

Reply:
We are not exporting `zh` in the tileset, because that is not a useful developer-facing choice of locale. Ideally we normalize `zh` in the raw data into both `zh-Hans` and `zh-Hant` if they are the same.